rinex-cli/src/positioning/ppp/post_process.rs

```diff
@@ -117,6 +113,27 @@ pub fn post_process(
     );
     plot_ctx.add_trace(trace);
 
+    let mut worst_radius = 1000.0_f64;
+    /*
+     * Add Spherical mesh with radius being the
+     * largest error
+     */
+    for error in results
+        .iter()
+        .map(|(_, pvt)| (pvt.pos.x.powi(2) + pvt.pos.y.powi(2) + pvt.pos.z.powi(2)).sqrt())
```

Comment:
Nice, looks like LLVM knows will transform `x.powi(2)` into `x * x`, I didn't know that.